lib/compile/bundles/bootstrap.js

```diff
@@ -0,0 +1,21 @@
+import most from "most";
+
+import resolveAsset from "../modules/resolve";
+import loadAst from "../modules/load-ast";
+
+export default function (compilation) {
+  return most
+    .from(compilation.opts.emit)
+    .filter(def => def.entry || def.split)
+    .map(function bootstrapBundle (bundleDef) {
+      const src = bundleDef.entry || bundleDef.split;
+      const asset = resolveAsset.call(compilation, src);
```

Comment:
Agreed - I'm going to clean this up.  Originally I wanted to avoid passing an additional argument everywhere - but ultimately, I'm doing it anyway, and in a much more opaque way.